meson.build

```diff
@@ -22,6 +22,9 @@ else
 endif
 conf.set('ENABLE_USE_BUFFER_HEADER', get_option('USE_BUFFER_HEADER'))
 
+cc = meson.get_compiler('c')
+execinfo_dep = cc.find_library('execinfo', required : false)
```

Comment:
This dependency is probably required on freebsd (and we should not even test is on other platform).

We should find the library only if we are on freebsd (this can help : https://mesonbuild.com/Reference-tables.html#operating-system-names)